token-bridge/README.md

```diff
@@ -0,0 +1,297 @@
+
+# POA TokenBridge
+
+[![Build Status](https://travis-ci.org/poanetwork/token-bridge.svg)](https://travis-ci.org/poanetwork/token-bridge)
```

Comment:
Does it make sense to remove the TravisCI badge and move the gitter badge to the README file in the root directory of the repo?

Reply:
Addressed by https://github.com/poanetwork/tokenbridge/pull/19